app/models/user.rb

```diff
@@ -884,6 +885,10 @@ def compact_uploader?
     post_upload_count >= 10 && enable_compact_uploader?
   end
 
+  def use_new_search_syntax?
+    use_new_search_syntax
+  end
```

Comment:
You don't need to define this, the HasBitFlags class already defines it
https://github.com/e621ng/e621ng/blob/3229106431dbf2636a4fb7e2586cc3ef7da5bf57/app/logical/danbooru/has_bit_flags.rb#L26-L28

Reply:
Fixed 276b298b64e86871fbab7b131b865f9700a3d8a0